tests/apis.js

```diff
@@ -1,88 +1,54 @@
 import test from 'ava';
-import uuid from 'uuid';
-import ipsum from 'lorem-ipsum';
-import Promise from 'bluebird';
 import slugify from 'underscore.string/slugify';
 import isEqual from 'lodash/isEqual';
 import cloneDeep from 'lodash/cloneDeep';
+import isUUID from 'validator/lib/isUUID';
+
 import apiUtils from '../lib/api/utils';
 import api from '../lib/api';
-import utils from '../lib/utils';
 import database from '../lib/database';
+import merged from './fixtures/live/objects/types-models-merged';
+import dbmocks from './fixtures/live/objects/database-mocks.js';
 
-const content = [];
-const types = [];
-const eachOfType = {};
-const allTypes = [];
-
-for (let i = 0; i < 5; i++) {
-  types.push(`Test Type ${ipsum({
-    count: 1,
-    units: 'words',
-  })}`);
-  eachOfType[types[i]] = 0;
-  allTypes.push({
-    name: types[i],
-    id: slugify(types[i]),
-    description: ipsum({
-      count: Math.round(Math.random() * 6 + 1),
-      units: 'words',
-    }),
-    attributes: [],
-  });
-}
+const content = dbmocks.rows;
+const types = merged.types;
+const eachOfType = dbmocks.eachOfType;
+const allTypes = merged.allTypes;
 
-for (let i = 0; i < 50; i++) {
-  const name = ipsum({
-    count: 3,
-    units: 'words',
-    format: 'plain',
+test.cb.before(t => {
+  database.init().then(() => {
+    database('live').insert(content).then(() => {
+      t.end();
+    }).catch(e => { // because we can't return in a before, this catch doesn't bubble out
+      t.fail(e);
+    });
+  }).catch(e => {
+    t.fail(e.message);
   });
+});
 
-  const sunrise = `${Math.round(Math.random() * 3 + 2016)}-${Math.round(Math.random() * 11 + 1)}-${Math.round(Math.random() * 25 + 1)}`;
-  const sunset = `${Math.round(Math.random() * 3 + 2016)}-${Math.round(Math.random() * 11 + 1)}-${Math.round(Math.random() * 25 + 1)}`;
-
-  const type = types[Math.round(Math.random() * 4)];
-
-  eachOfType[type] += 1;
-
-  const item = {
-    'id': uuid.v4(),
-    'language': 'en-us',
-    'sunrise': utils.time.iso(sunrise, '00:00'),
-    'sunrise-timezone': 'America/New_York',
-    'sunset': utils.time.iso(sunset, '00:00'),
-    'sunset-timezone': 'America/New_York',
-    'key': name,
-    'key-slug': slugify(name),
-    type,
-    'type-slug': slugify(type),
-    'attributes': {
-      name,
-      text: ipsum({
-        count: 3,
-        units: 'paragraphs',
-        format: 'html',
-      }),
-    },
-  };
-
-  content[i] = item;
-}
+//////////////////////////////
+// Utils - attributes
+//////////////////////////////
+test('Utils: attributes', t => {
+  const item = Math.round(Math.random() * content.length);
```

Comment:
This may be a source of errors. If `Math.random() === 1`, then `item` will equal `length`, and the next line will fail because `content[lenght]` doesn't exist.